src/main/java/org/spongepowered/api/entity/Entity.java

```diff
@@ -560,4 +561,17 @@ default boolean canSee(Entity entity) {
         return getValue(Keys.HAS_GRAVITY).get();
     }
 
+    /**
+     * Gets the {@link SoundType} that is played when this entity moves in water.
+     *
+     * @return The SoundType for noise this entity makes when it swims
```

Comment:
IMO these sentences could be simplified to:
````
The sound that the entity emits when it ...
````
Same goes for the other methods. Not 100% sure though.